docusaurus-docs/conda-store/how-tos/install-standalone.md

```diff
@@ -51,10 +51,9 @@ Access conda-store UI at [`localhost:8080`](https://localhost:8080/) 🎉
 
 ## Next steps
 
-Learn to use the [conda-store UI with these tutorials][conda-store-ui-tutorials].
+Learn more about [conda-store ui][conda-store-ui].
 
 
 <!-- Internal links -->
 
 [conda-store-ui]: ../../conda-store-ui/introduction
-[conda-store-ui-tutorials]: ../../conda-store-ui/tutorials
```

Comment:
Is there a reason to remove the tutorials?

Reply:
oops, over edit. Thanks, will add these back.